crates/polars-python/src/lazyframe/visit.rs

```diff
@@ -57,7 +57,7 @@ impl NodeTraverser {
     // Increment major on breaking changes to the IR (e.g. renaming
     // fields, reordering tuples), minor on backwards compatible
     // changes (e.g. exposing a new expression node).
-    const VERSION: Version = (3, 2);
+    const VERSION: Version = (4, 0);
```

Comment:
I don't think this is a breaking IR change, but a minor version bump.

Reply:
We changed the structure of the tuple in the join options (one more field)